faq.md

```diff
@@ -0,0 +1,153 @@
+IP-based tracking: [Willful IP blindness](https://github.com/bslassey/ip-blindness).
+
+### How will the privacy budget's assessment of the "cost" of APIs change over time? On what basis?
+
+_API "cost" = How much of the privacy budget would be spent when a site uses the API._
+
+Identifiability of APIs and features will change with time as browser / device ecosystems and user
+populations change. In the long term, browsers will need to periodically adjust the costing strategy
+to keep up, which will in turn require periodic surveys. How and who will do this is not determined
+at this time. Any resulting changes to the costing strategy will be handled in the same way as an
+API change.
+
+### Will the privacy budget make web APIs less reliable? Will the APIs I'm using be constrained tomorrow even though I'm within the budget today?
+
+_API "cost" = How much of the privacy budget would be spent when a site uses the API._
+
+Since an API's cost would affect its availability, it may effectively become part of the API
+surface. In this case: when an API's cost would change, the same process as for any other API change
+would need to be followed (public announcements, appropriate reviews...). Note: an API's cost change
+may be induced by an API change itself (e.g. if the API, for the sake of enabling more capabilities,
+accesses more browser information) or in more rare cases by browsers detecting that an API is more
+identifying than previously evaluated.
+
+### How will browser vendors maintain transparency and consistency over the privacy budget's costing strategy?
+
+This is an open question, and it will need to be publicly established in the future. It depends on
+the approach different browser vendors decide to take.
+
+### Is the privacy budget feasible?
+
+This proposal is at an early stage and exact privacy budget limits are to be determined. But there
+are reasons to think that a privacy budget is feasible: it takes about **32 bits of entropy** to
+uniquely identify people on the web, but mobile web users (which make up most of web users) can only
+expose at most about **13 bits** of entropy.
+
+About these numbers:
+
+- There are about [4.6 billion of web
+  users](https://www.statista.com/statistics/617136/digital-population-worldwide/). So it takes
+  log2(4.6 billion) = about 32 bits of entropy to uniquely identify people on the web.
+- Mobile web users make up [most of web
+  users](https://gs.statcounter.com/platform-market-share/desktop-mobile-tablet/).
+- There's an order of 10,000 different mobile device models (estimates range from 4,000 to 60,000+).
+  Devices of the same model have the same fingerprint. So mobile web users can only expose at most
+  log2(10,000) = about 13 bits of entropy. Note that the mobile device population is very unevenly
+  distributed over the thousands of device models, so the entropy of knowing a mobile device model
+  may be significantly lower than 13 bits.
```

Comment:
```suggestion
```

Reply:
This is a rough estimate of how much entropy the device model exposes, although as it notes it is not an even distribution. The translation into "but mobile web users can only expose 13 bits" isn't right. **If** a site requests the device model and device model were to be measured as exposing 13 bits (which seems like an over estimate, but leave that aside for now) then those 13 bits would be deducted from the budget. So, if the budget were, say, 28 bits there would be 15 bits remaining in the budget, not 13.